src/documents/yesNo.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Yes/No
+layout: default
+category: API
+---
+
+## Overview
+
+**Component Name:** `gpii.firstDiscovery.panel.yesNo`
+
+**File:** `panels.js`
+
+Provides a means to create panels that select and only select one of two choices, such as yes or no.
+This component is not intended to be used on its own, but provided as a base grade to another
+component that will use it's capabilities for creating such panels.
+An UI example of one of these panels:
+
+![A question with two buttons below it, labelled "yes" and "no"](images/yesNoPanel.jpeg)
+
+## Using the Yes/No grade
+
+To use the Yes/No grade, supply it as a `gradeNames` option in your component definition:
+```javascript
+fluid.defaults("my.component", {
+    gradeNames: ["gpii.firstDiscovery.panel.yesNo"],
+    ...
+});
+```
+
+## Grades
+
+This component uses the following base
+[grades](http://docs.fluidproject.org/infusion/development/ComponentGrades.html):
+
+* [`gpii.firstDiscovery.attachTooltip.renderer`](attachTooltipRenderer.md)
+* [`fluid.prefs.panel`](http://docs.fluidproject.org/infusion/development/Panels.html)
+
+## Model
+
+This component supports the following
+[model](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ModelComponents.html)
+properties:
+
+| Path   | Description | Values | Default |
+|--------|-------------|--------|---------|
+| `value` | The corresponding boolean value of the selected choice. `true` when the selected choice is yes; `false` when the selected choice is no. | Boolean | undefined |
+| `choice` | The selected choice. | String | undefined |
+
+
+## Options
+
+This component can be configured using the following
+[options](http://docs.fluidproject.org/infusion/development/ComponentOptionsAndDefaults.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`controlValues`</td>
+        <td>An object that contains a "choice" element that determines choice values assigned to HTML radio buttons.</td>
+        <td>Object<br/>"choice" value is an array with two elements.</td>
```

Comment:
Probably add a period after "Object" to make clear this word is on its own. Right now, when being rendered as html, "Object" looks like a part of the following sentence, which adds confusion.